packages/fxa-settings/src/index.tsx

```diff
@@ -39,26 +41,53 @@ try {
     return document.head.querySelector(name);
   });
 
-  const apolloClient = createApolloClient(config.servers.gql.url);
-  const appContext = initializeAppContext();
+  const completeInitialization = () => {
+    const apolloClient = createApolloClient(config.servers.gql.url);
+    const appContext = initializeAppContext();
 
-  render(
-    <React.StrictMode>
-      <AppErrorBoundary>
-        <AppContext.Provider value={appContext}>
-          <AppLocalizationProvider
-            baseDir="/settings/locales"
-            userLocales={navigator.languages}
-          >
-            <ApolloProvider client={apolloClient}>
-              <App {...{ flowQueryParams }} />
-            </ApolloProvider>
-          </AppLocalizationProvider>
-        </AppContext.Provider>
-      </AppErrorBoundary>
-    </React.StrictMode>,
-    document.getElementById('root')
-  );
+    render(
+      <React.StrictMode>
+        <AppErrorBoundary>
+          <AppContext.Provider value={appContext}>
+            <AppLocalizationProvider
+              baseDir="/settings/locales"
+              userLocales={navigator.languages}
+            >
+              <ApolloProvider client={apolloClient}>
+                <App {...{ flowQueryParams }} />
+              </ApolloProvider>
+            </AppLocalizationProvider>
+          </AppContext.Provider>
+        </AppErrorBoundary>
+      </React.StrictMode>,
+      document.getElementById('root')
+    );
+  };
+  if (
+    config.sendFxAStatusOnSettings &&
+    shouldSendFxAStatus(flowQueryParams.context)
```

Comment:
@LZoog I think this makes sense to use the OAuth integration and check for context, do we have an example I can point too? If not maybe it is ok to land this by checking the flow query param and then fix in follow up.